docs/content/en/docs/references/yaml/_index.html

```diff
@@ -7,6 +7,10 @@
     weight: 30
 ---
 
+{{< alert title="🚀 Tooltip" >}}
```

Comment:
"Tooltip" seems an odd choice?  "Tip"?

I've seen the use of the lightbulb for tips elsewhere: <kbd>💡  Tip</kbd>